src/components/navigation/Navbar.scss

```
@@ -0,0 +1,148 @@
+@import '../../constants.scss';
+
+.emptySpacingBox{
+  width:100%;
+  height:50px;
+}
+
+#menuToggle
+{
+  display: block;
+  position: fixed;
+  top: 20px;
+  left: 20px;
+  height:100%;
+  z-index: 1;
+  -webkit-user-select: none;
+  user-select: none;
+}
+
+#menuToggle a
+{
+  text-decoration: none;
+  color: white;
+  transition: color 0.3s, underline 0.3s;
+}
```

We're using SCSS, so all of these can be grouped/nested for clarity!
```suggestion
  user-select: none;
  
  a  {
    text-decoration: none;
    color: white;
    transition: color 0.3s, underline 0.3s;
  }
}
```